lib/chezmoi/scripts.go

```diff
@@ -0,0 +1,97 @@
+package chezmoi
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+	"regexp"
+)
+
+var shebangRegex *regexp.Regexp
+var onceRegex *regexp.Regexp
+
+func init() {
+	shebangRegex = regexp.MustCompile(`(?m)^#!([^;\n]+)$`)
+	onceRegex = regexp.MustCompile(`(?m)^#\s*chezmoi:\s*once$`)
+}
```

Comment:
The rexgexps can be initialized where they are declared, no need for an `init` function.